### PR TITLE
Update migrating-to-uno-5.md

### PR DESCRIPTION
**Pull request by EvarDion**
Add fix to the doco for migrating from shared class libraries form .net7 to 8

GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/migrating-to-uno-5.md [hunk 1]
```diff
@@ -49,6 +49,17 @@ Hot Reload support has changed in Uno Platform 5.0 and a new API invocation is n
 
 Note that Hot Reload has changed to be based on C#, which means that changes done XAML files will need the use of C# Hot Reload feature to be applied to the running app. See [this documentation](xref:Uno.Features.HotReload) for more details.
 
+#### Migrating Shared Class Libraries from net7.0 to net8.0
+
+- If you are building on windows and experience the compilation error NETSDK1083 when retargetting your library from net7.0 to net8.0, add the following to the `csproj`:
+    ```xml
+    <PropertyGroup>
+        <RuntimeIdentifiers>win-x64;win-x86</RuntimeIdentifiers>
```
Comment (dansiegel) [inline doc/articles/migrating-to-uno-5.md:57]:
```suggestion
        <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
```

**Comment by jeromelaban** [comment]
Thanks for your contribution! Could you change squash your commits and use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary)? something like `docs: Update migrating to uno 5`.

**Comment by EvarDion** [comment]
@jeromelaban, done I think... I'm fairly new to using Git web and desktop app.

**Comment by jeromelaban** [comment]
No worries, we've got a small guide here on how to do this: https://platform.uno/docs/articles/contributing/guidelines/pull-requests.html

**Pull request closed without merging**